File: api/crypto/src/main/java/net/corda/v5/crypto/merkle/MerkleProofRebuildFailureException.java
package net.corda.v5.crypto.merkle;

import net.corda.v5.base.exceptions.CordaRuntimeException;
import org.jetbrains.annotations.NotNull;

/**
 * Indicates that the calculation of the root hash of a {@link MerkleProof} failed.
 */
public final class MerkleProofRebuildFailureException extends CordaRuntimeException {
    public MerkleProofRebuildFailureException(@NotNull String message) {
        super(message, null);
    }
}

File: crypto/src/main/kotlin/net/corda/v5/crypto/merkle/MerkleProofRebuildFailureException.kt
package net.corda.v5.crypto.merkle

import net.corda.v5.base.exceptions.CordaRuntimeException

/**
 * Indicates that the calculation of the root hash of a [MerkleProof] failed.
 */
class MerkleProofRebuildFailureException(message: String) : CordaRuntimeException(message, null)